data/migrations/20181024004304_customer.js

```diff
@@ -1,12 +1,9 @@
 
 exports.up = (knex, Promise) => {
     return knex.schema.createTable('customer', (table) => {
-        table.increments();
-        table.string('username').unique().notNullable();
-        table.string('externalID').unique().notNullable();
+        table.string('externalID').unique().notNullable().primary();
```

Comment:
Is `externalID` a good name for this field? Or should we just change to `id`?

Reply:
I think externalID is fine